client/web/src/components/app.tsx

```diff
@@ -112,9 +113,9 @@ function Header({
         </div>
         <LoginToggle node={node} auth={auth} newSession={newSession} />
       </div>
-      {loc !== "/" && (
+      {loc !== homeURL && loc !== homeURL + "/" && (
```

Comment:
Oh does this not pick up on the base we set?

Reply:
apparently not.  On my Synology, it was linking me back to the root without the base

Reply:
Ah, I think it's because our <Header> is outside our <Router>.  Lemme try moving that.

Reply:
there we go... that's much better.